objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/imageRetriever/NetworkStreamRetriever.java

```diff
@@ -68,8 +70,8 @@ public void setupDataRetrieval(Map<String, ?> dataSourceConfig, ObjectRecognitio
         }
         if (!capture.isOpened()) {
             diagnoseConnection();
-//            throw new IllegalArgumentException(this.getClass().getCanonicalName() + ".notVideoStream: " 
-//                    + "URL does not represent a video stream");
+            throw new IllegalArgumentException(this.getClass().getCanonicalName() + ".notVideoStream: " 
+                    + "URL does not represent a video stream");
```

Comment:
Better wording might something like "Stream at URL cannot be opened as a video stream."